core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/federation/TupleFunctionFederatedService.java

```diff
@@ -137,12 +137,8 @@ public void remove() throws QueryEvaluationException {
 			}
 
 			@Override
-			public void handleClose() throws QueryEvaluationException {
-				try {
-					super.handleClose();
-				} finally {
-					iter.close();
-				}
```

Comment:
I have started removing calls to super methods where the super method is just an empty method. In the future i would actually like to remove these empty methods.